docker-app/qfieldcloud/subscription/models.py

```diff
@@ -31,6 +31,7 @@ def get_or_create_default(cls) -> "AccountType":
     can_add_storage = models.BooleanField(default=False)
     can_add_job_minutes = models.BooleanField(default=False)
     is_external_db_supported = models.BooleanField(default=False)
+    is_nongpkg_supported = models.BooleanField(default=False)
```

Comment:
@suricactus Am I correct understanding `is_external_db_supported` and the new `is_nongpkg_supported` are unrelated ?

Reply:
Yes they are pretty much the same.